gcloud/storage/test_batch.py

```diff
@@ -101,92 +101,108 @@ def test_ctor_w_implicit_connection(self):
 
         self.assertTrue(batch._connection is connection)
         self.assertEqual(len(batch._requests), 0)
-        self.assertEqual(len(batch._responses), 0)
+        self.assertEqual(len(batch._futures), 0)
 
-    def test__make_request_GET_forwarded_to_connection(self):
+    def test__make_request_GET_normal(self):
+        from gcloud.storage.batch import _FutureDict
         URL = 'http://example.com/api'
         expected = _Response()
         http = _HTTP((expected, ''))
         connection = _Connection(http=http)
         batch = self._makeOne(connection)
-        response, content = batch._make_request('GET', URL)
-        self.assertTrue(response is expected)
-        self.assertEqual(content, '')
+        target = _MockObject()
+        response, content = batch._make_request('GET', URL,
+                                                target_object=target)
+        self.assertEqual(response.status, 204)
+        self.assertTrue(isinstance(content, _FutureDict))
+        self.assertEqual(target._properties, content)
```

Comment:
@tseaver comment from cbabbb1b44ac87e84b7932adc1b3e87a711c4ecf

> Should this assert equality or identity?

----

Identity, though it's fairly irrelevant, no?